src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/custom.py

```diff
@@ -162,6 +162,27 @@ def tag_resource(tags, resource_group_name=None, resource_provider_namespace=Non
                          resource_id, api_version)
     return res.tag(tags)
 
+def get_deployment_operations(client, resource_group_name, deployment_name, operation_id=None,
+                              operation_ids=None):
+    """Gets a deployments operation.
+    :param resource_group_name: The name of the resource group. The name
+     is case insensitive.
+    :type resource_group_name: str
+    :param deployment_name: The name of the deployment.
+    :type deployment_name: str
+    :param operation_ids: A comma separated list of ids to get.
+    :type operation_ids: str
+    """
+    if not operation_id and not operation_ids:
```

Comment:
if we only expose `operation_ids`, this if condition can be removed. Same with the next if